Pong.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "DylanAkp",
+			"oem": "Nothing",
+			"device": "Nothing Phone (2)",
+			"filename": "crDroidAndroid-14.0-20240207-Pong-v10.1.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/Pong/10.x/crDroidAndroid-14.0-20240207-Pong-v10.1.zip/download",
+			"timestamp": 1707281856,
+			"md5": "9edca70ce8707a94955adb0b84bc69f4",
+			"sha256": "b67adce3a557d932ec16ffdbd3f2236d6ed91e89882869df9996583278a73448",
+			"size": 1739386762,
+			"version": "10.1",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/4646393/",
+			"gapps": "https://nikgapps.com/crdroid-official",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "https://www.paypal.com/paypalme/dylanakp",
+			"telegram": "https://t.me/+bsJmd23ZEy1jMTg0",
+			"dt": "https://github.com/crdroidandroid/android_device_nothing_Pong",
+			"common-dt": "",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_nothing_sm8475"
```

Comment:
sources not pushed

Reply:
Kernel sources has been pushed and everything is now up to date